pydicom/tests/test_getdata.py

```diff
@@ -0,0 +1,33 @@
+"""Test the data manager"""
+
+import os
+import unittest
+
+from pydicom.data import (get_charset_files,
+                          get_testdata_files)
+
+from pydicom.data.base import DATA_ROOT
+
+
+class TestGetData(unittest.TestCase):
+
+    def test_get_dataset(self):
+        """Test the different functions
+        to get lists of data files"""
+
+        # Test base locations
+        charbase = os.path.join(DATA_ROOT, 'charset_files')
+        self.assertTrue(os.path.exists(charbase))
+
+        testbase = os.path.join(DATA_ROOT, 'test_files')
+        self.assertTrue(os.path.exists(testbase))
+
+        # Test file get
+        chardata = get_charset_files()
+        self.assertTrue(len(chardata) > 15)
```

Comment:
I would check something different, like that the filenames of the files are the same, or some checksum or something. In this manner if we change the files by accident the test will fail.